Completed first draft of the Motivation Episode (#24)
carpentries-incubator/managing-computational-projects: _episodes/02-motivation.md

- "First key point. Brief Answer to questions. (FIXME)"
---

> ## Why are we thinking about coding in biomedical and life sciences?

Reviewer: ```suggestion
## Why Are We Thinking about Computation and Data Science in Biomedical and Life Sciences?
```
Camel casing title and replacing code with umbrella terms

> <img src="https://zenodo.org/record/3678226/files/ReproducibleJourney.jpg" alt="drawing" width="500"/>

Reviewer: ```suggestion
<img src="https://zenodo.org/record/3678226/files/ReproducibleJourney.jpg" alt="drawing" width="500"/>

_What to expect in your reproducibilty journey. The Turing Way project illustration by Scriberia for The Turing Way Community Shared under CC-BY 4.0 License. Zenodo. http://doi.org/10.5281/zenodo.3332807_
```

Reviewer: Can you please annotate each image (alt text and title with references)?

Writing, using, and sharing code and data within biomedical and life sciences is growing rapidly. Software is not only the realm of big tech and startups, with more and more scientists becoming programmers as part of their workflow. From computational power comes greater calculations, modelling, visualisations, and analyses, and therefore expanding our insights and methodologies. Regardless of the tools and languages we use, coding is integral to science. 

<img src="https://media.nature.com/lw800/magazine-assets/d41586-021-00075-2/d41586-021-00075-2_18964266.jpg" alt="drawing" width="300"/> <img src="https://media.nature.com/lw800/magazine-assets/d41586-021-00075-2/d41586-021-00075-2_18880300.jpg" alt="drawing" width="288"/>

> > *Nature 589, 344-348 (2021) https://doi.org/10.1038/d41586-021-00075-2*

Reviewer: ```suggestion
> *Nature 589, 344-348 (2021) https://doi.org/10.1038/d41586-021-00075-2*
```
Is this the right reference?
